contracts/ens-label-healer/.env.example

```
@@ -0,0 +1,15 @@
+
+# Address that receives SUBMITTER_ROLE during deployment (optional).
+# If omitted, grant it manually after deployment via grantRole.
+SUBMITTER_ADDRESS=
+
+# Private key of the deployer account (required for deployment).
+# Must hold enough ETH to cover gas on the target network.
+DEPLOYER_PRIVATE_KEY=
+
+# RPC URLs for each network.
+SEPOLIA_RPC_URL=
+MAINNET_RPC_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MAINNET_RPC_URL key should go before the SEPOLIA_RPC_URL key

(UnorderedKey)